0000-template.md

```diff
@@ -1,7 +1,11 @@
 
-- Start Date: (fill me in with today's date, YYYY-MM-DD)
-- RFC PR: (leave this empty)
-- FOLIO Issue: (leave this empty)
+- Stage: (DRAFT REVIEW, PRELIMINARY REVIEW, PUBLIC REVIEW, FINAL REVIEW)
```

Comment:
The branching strategy I would propose could mitigate the need for this field. It seems weird to update this field, after every stage if only the final version is what is saved in this repo.

Reply:
good point.  We can hold off on this until we see your proposal.  It's obviously important that the two align

Reply:
Agreed, given that the branches have the potential to "automatically" update the status depending on how they are labeled, it would be great to not have to rely on people remembering to update the field.

Reply:
The stage field is redundant given that we have all the stages enumerated below in the RFC PRs section.  It will be clear which stage the RFC is in.